packages/cli-lib/templates.js

```diff
@@ -1,31 +1,36 @@
 const fs = require('fs');
+const { logger } = require('./logger');
 
 // Matches the .html file extension, excluding module.html
 const TEMPLATE_EXTENSION_REGEX = new RegExp(/(?<!module)\.html$/);
 // Matches the comment brackets that wrap annotations
 const ANNOTATIONS_REGEX = /<!--([\s\S]*?)-->/;
 // Matches an annotation value, ending at space, newline, or end of string
-const ANNOTATION_VALUE_REGEX = ':\\s?([\\S]*?)(\\s|\n|$)';
+const ANNOTATION_VALUE_REGEX = ':\\s?([\\S|\\s]*?)(\n|$)';
```

Comment:
Fixing this to support annotation values that have spaces in them. For example, before the changes, this label annotation was getting resolved as "My" because I had "space" as something that could end the grouping.
```html
<!--
  label: My label
-->
```